input/pagecontent/change_log.md

```diff
@@ -78,18 +78,26 @@ In addition, the following value sets are now associated with the non-TNM [Cance
   * #1162615005 "Lymph node level XA (qualifier value)"
   * #1162613003 "Lymph node level XB (qualifier value)"
 
-### Profile Categories
```

Comment:
Need to add the redesign of slicing with the Bundle to the change_log

Reply:
OK, done.